jetty-webapp/src/test/java/org/eclipse/jetty/webapp/WebAppContextTest.java

```diff
@@ -296,18 +296,6 @@ public void testNullSessionAndSecurityHandler() throws Exception
         assertTrue(context.isAvailable());
     }
 
-    @Test
-    public void ordering() throws Exception
```

Comment:
Why has this ordering test been removed?  WEB-INF/lib jars definitely have an order and we need to test it.

Reply:
No ClassLoader has that guarantee.
WebAppClassLoader extends from URLClassLoader which also has no guarantee of search order within it.

Reply:
Will restore, but change the name of the test method.

`s/ordering/testMetadataRegisteredOrder/`

Reply:
I'm kind of convinced that this is not a necessary (or meaningful) test.
So long as the classloader is always fed ordered jars (so we are deterministic and not dependent on the filesystem) and we test that order (in the fragment tests I think), then I'm happy

Perhaps we should deprecate or hide the unordered getter.

Reply:
This bad test is using `WebInfConfiguration` to test `MetaData` registered order.
But that's something determined by the web fragments.
Not what this specific test is assuming.
We do have tests for web fragment determined orders, `org.eclipse.jetty.webapp.OrderingTest`.
Which specifically tests `MetaData.getOrderedWebInfJars()` as intended.